tests/README.md

```diff
@@ -17,7 +17,7 @@ limitation can be lifted once we can select `footloose` containers better
 To run all tests:
 
 ```console
-go tests -v ./tests
+go test -v ./tests
```

Comment:
There are two `go tests` typos in the file, could you fix the other one while you're there :)